src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -39,13 +74,78 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
     [categories]
   );
 
+  const { relatedVenueIds } = useRelatedVenues({
+    //isLoading, relatedVenues,
```

Comment:
Why is this commented out here?

Reply:
cleaned up — from prior idea to show user avatar on poster preview. now removed